src-js/app.js

```diff
@@ -1,3 +1,77 @@
 import * as Cookie from 'js-cookie';
+import * as lodash from 'lodash.throttle';
```

Comment:
Could we rename this to `throttle` since that's all we're using here?
lodash is a pretty popular library, so I find it confusing to see `lodash(function (query, searchFunction) { searchFunction(query); }, 1000)`

Reply:
sure